stdlib/number.gr

```diff
@@ -86,6 +86,24 @@ export let rec sqrt = (x: Number) => {
   ret
 }
 
+/**
+ * Determine the positivity or negativity of a Number.
+ *
+ * @param x: The number to inspect
+ * @returns `-1` if the number is negative, `1` if positive, or +/-`0` otherwise
+ *
+ * @example Math.sign(-10000) == -1
+ * @example Math.sign(222222) == 1
+ * @example Math.sign(-0.0) == -0.0
```

Comment:
This is the Number library 😛 

```suggestion
 * @example Number.sign(-10000) == -1
 * @example Number.sign(222222) == 1
 * @example Number.sign(-0.0) == -0.0
```

Also, `Number.sign(-0.0) == 0.0`. Is worth having that example?

Reply:
🙈

Reply:
> Also, Number.sign(-0.0) == 0.0. Is worth having that example?

So we want both `Number.sign(-0.0) == -0.0` and `Number.sign(-0.0) == 0.0` docs? It almost like we're demo'ing the equality operator at that point 😛

Reply:
Oh, I meant should we _remove_ that example because it's just checking `0 == 0`.

Reply:
I'll just change it to `Number.sign(0) == 0` to demonstrate the 0 behavior then.